packages/ag-charts-community/src/chart/dom/domManager.ts

```diff
@@ -161,24 +171,37 @@ export class DOMManager extends BaseManager<Events['type'], Events> {
         return this.parentElement;
     }
 
-    setSize(autoSize?: boolean, optionsWidth?: number, optionsHeight?: number) {
+    setSizeOptions(minWidth: number = 0, minHeight: number = 300, optionsWidth?: number, optionsHeight?: number) {
         const { style } = this.parentElement.element;
 
-        style.width = optionsWidth != null ? `${optionsWidth}px` : '100%';
-        style.minWidth = optionsWidth != null || autoSize === true ? '' : '300px';
-        style.height = optionsHeight != null ? `${optionsHeight}px` : '100%';
-        style.minHeight = optionsHeight != null || autoSize === true ? '' : '300px';
+        style.minWidth = `${optionsWidth ?? minWidth}px`;
+        style.minHeight = `${optionsHeight ?? minHeight}px`;
```

Comment:
This should be equivalent to

```suggestion
        style.width = `${optionsWidth ?? minWidth}px`;
        style.height = `${optionsHeight ?? minHeight}px`;
```

The advantage of explicit sizes are the browser can optimise its layout calculations better

Reply:
Tested it out and you are right - fixed.